apps/remote_control/test/lexical/remote_control/code_intelligence/entity_test.exs

```diff
@@ -843,6 +850,15 @@ defmodule Lexical.RemoteControl.CodeIntelligence.EntityTest do
       assert {:ok, {:module_attribute, Parent, :foo}, resolved_range} = resolve(code)
       assert resolved_range =~ "%{foo: «@foo»}"
     end
+
+    test "returns a failure if you're not in a module" do
```

Comment:
I'm not quite sure what "returns a failure" means in this context, especially since it's still returning `{:ok, something}`. Perhaps this test could be "resolves modules as nil when not in a module"?

Reply:
I initially made this a failure case, and didn't update the test message.